masu/util/common.py

```diff
@@ -48,3 +54,14 @@ def stringify_json_data(data):
         return str(data)
 
     return data
+
+
+def ingest_method_for_provider(provider):
```

Comment:
This feels to me like it should be defined as either a config value or an attribute of the provider class.

Can you help me understand why it makes sense to define these values here?

Reply:
Do you have a suggestion of what provider class?  It's just a simple map for the orchestrator to know what provider types should be polled.  I could see it living in the orchestrator itself but figured putting it as a common utility made sense so it could be used in other parts of the code.

Reply:
I was thinking each Provider class defines its own ingestion method. It could be a class variable or a module-level constant.

Reply:
I was thinking each Provider class defines its own ingestion method. It could be a class variable or a module-level constant.

To me, this looks like configuration data. But, rather than having each provider own it's piece of data, you've centralized all values into a single spot.

The problem is more organizational than technical - it's unclear why this data needed to be centralized when each provider owns other aspects of its own config.